### PR TITLE
Typos, links, and small style fixes

### LINKED ISSUE #146: [DOC] - Typo on banner
### Preliminary Checks

- [X] This issue is not a question, feature request, RFC, or anything other than a bug report. Please post those things in GitHub Discussions: https://github.com/nebari-dev/nebari/discussions

### Summary

The banner at the top of the page has a capitalization typo. 

`We are currently undergoing a rename from [QHUb](https://docs.qhub.dev/)`

should be 

`We are currently undergoing a rename from [QHub](https://docs.qhub.dev/)`

(The `u` in `QHub`)

### PR DESCRIPTION
**Pull request by pavithraes**
## Reference Issues or PRs
- Fixes #146

## What does this implement/fix?

_Put an `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features to not work as expected)
- [x] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [x] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [x] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [x] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [x] This content adheres to the Nebari style guides.

#### Non-text content

- [ ] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

**Review by trallard — APPROVED** [review]
Thanks for opening this 🙏🏽

File: docs/docs/how-tos/domain-registry.md [hunk 1]
```diff
@@ -78,7 +78,10 @@ The amount of time this takes varies for each DNS provider. Validate such inform
 
 Qhub has an extra flag for deployments that grants management and the creation of the DNS records for you automatically. For automatic DNS provision add `--dns-auto-provision` to your qhub deploy command:
 
-`qhub deploy -c qhub-config --dns-provider cloudflare --dns-auto-provision`
+```console
+qhub deploy -c qhub-config.yaml --dns-provider cloudflare --dns-auto-provision
```
Comment (trallard) [inline docs/docs/how-tos/domain-registry.md:82]:
```suggestion
qhub deploy -c qhub-config.yaml \
  --dns-provider cloudflare \
  --dns-auto-provision
```

making it easier to read in small screens

**Review by trallard — APPROVED** [review]
LGTM - there is only one suggestion that needs committing, but after that, you can self-merge @pavithraes 

Thanks a lot!

File: docs/docs/how-tos/domain-registry.md (hunk 1 above)
Comment (pavithraes) [reply docs/docs/how-tos/domain-registry.md:82]: Thank you!

**Pull request merged**